src/modules/office/OfficeState.js

```diff
@@ -0,0 +1,98 @@
+import {Map} from 'immutable';
+import {loop, Effects} from 'redux-loop-symbol-ponyfill';
+import {NavigationActions} from 'react-navigation';
+import {getPlace, getInfoPlace, getAnotherPlace} from '../../services/locationService';
+import {Alert} from 'react-native';
+
+// Initial state
+const initialState = Map({
+  position: 0,
+  loading: false,
+  value: {},
+  place: {}
+});
+
+// Actions
+const POSITION_CHANGED = 'POSITION_CHANGED';
+const REQUEST_OFFICE = 'REQUEST_OFFICE';
```

Comment:
The naming of some of these is a bit confusing, we generally try to keep it consistent with:
`REQUEST_OFFICE` and `RESPONSE_SUCCESS_OFFICE` or the other way around: `OFFICE_REQUEST` and `OFFICE_RESPONSE_SUCCESS` and `OFFICE_RESPONSE_FAILURE`